frontends/ol-components/src/components/RoutedDrawer/RoutedDrawer.tsx

```diff
@@ -80,11 +81,22 @@ const RoutedDrawer = <K extends string, R extends K = K>(
     window.history.pushState({}, "", `?${newParams}${hash || ""}`)
   }, [searchParams, params])
 
+  if (
+    process.env.NODE_ENV === "development" ||
+    process.env.NODE_ENV === "test"
+  ) {
+    const msg =
+      "[Dev-only Warning]: RoutedDrawer requires an aria-label or aria-labelledby prop."
+    invariant(props["aria-label"] || props["aria-labelledby"], msg)
+  }
+
   return (
     <Drawer
       open={open}
       onTransitionExited={removeUrlParams}
       onClose={setOpen.off}
+      role="dialog"
```

Comment:
So, I dunno why MUI doesn't put role=dialog on drawer by default. 

Bootstrap does, github does it on its drawers, and it resolves the accessibility checker warning because 
- https://github.com/dequelabs/axe-core/issues/991

Additionally, the drawer title is now read aloud when users open it, which is a big improvement.